pkgs/development/python-modules/qtwidgets/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pyqt5,
+  setuptools-scm,
+}:
+buildPythonPackage {
+  pname = "qtwidgets";
+  version = "1.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "pythonguis";
+    repo = "python-qtwidgets";
+    rev = "89bd9dc53990f04a82bb27cb056f0c513e40e8b0";
+    sha256 = "sha256-IoxNkuDR0jBh4EsTAzOBPAr5b4wEH5rLw2FAPSZj8Zo=";
+  };
+
+  patches = [
+    ./fix-qtgui-import.patch
+    ./fix-missing-svgs.patch
+  ];
+
+  build-inputs = [ setuptools-scm ];
```

```suggestion
  # Remove install_requires on markdown. It is not used.
  postPatch = ''
    substituteInPlace setup.py --replace-fail "'markdown'," ""
  '';

  build-system = [ setuptools-scm ];
```
Or amend the second patch

---

Whats the reasoning for this?

---

`pyproject = true` is the preferred build method for Python packages. In this case, the package states a dependency on markdown that makes no sense (it is not used anywhere).
So remove the dependency declaration.